recipes/json5/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "json5" %}
+{% set version = "0.8.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1ebf42c4de946a5e3e8bb348115b413f46be55d97fadda985c21ce795fb31196"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv "
+  entry_points:
+    - pyjson5 = json5.tool:main
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - json5
+
+about:
+  home: https://github.com/dpranke/pyjson5
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
```

Comment:
Raised upstream issue ( https://github.com/dpranke/pyjson5/issues/25 ) about getting the license file included in the package.

Reply:
Thanks @jakirkham